lib/fastlane/plugin/firebase_app_distribution/actions/firebase_app_distribution_action.rb

```diff
@@ -84,6 +84,12 @@ def self.available_options
                                        verify_block: proc do |value|
                                          UI.user_error!("firebase_app_distribution: Couldn't find ipa file at path '#{value}'") unless File.exist?(value)
                                        end),
+          FastlaneCore::ConfigItem.new(key: :gsp_path,
```

Comment:
Rather than abbreviate GSP, consider expanding this out to be called `googleservice_info_plist_path`; this name is a bit more intention revealing and doesn't require memorizing a new acronym.  It will also hopefully make it easier to grep for those doing exploratory review and maintain consistency with the environment variable naming I suggested.

Reply:
> Rather than abbreviate GSP, consider expanding this out to be called `googleservice_info_plist_path`; this name is a bit more intention revealing and doesn't require memorizing a new acronym. It will also hopefully make it easier to grep for those doing exploratory review and maintain consistency with the environment variable naming I suggested.

I initially picked `gsp_path` because it was already used in upload_symbols_to_crashlytics but I will make the suggested changes.